src/VerifyDeletionModal/VerifyDeletionModal.js

```diff
@@ -1,12 +1,29 @@
 import './VerifyDeletionModal.scss'
+import { deleteSingleItem } from '../apiCalls'
+
+const VerifyDeletionModal = ({ setVerifyDeletion, deleteItem, closeModal, categoryToDelete, itemToDelete, itemIdToDelete }) => {
+  const compileDeletionData = () => {
+    return {
+      data: {
```

Comment:
If we do an implicit return here, we can cut 7 lines of code down in here I think.

Reply:
Agreed!